Eco-Mode-WP/Eco-Mode
# Set up PHPUnit test framework and adds unit tests

Test instructions:
- Run `composer install` to install PHPUnit.
- Run `composer test` to run unit tests.
- Make sure that the (single) test passes.

Fixes #51
